core-bundle/src/Routing/Page/RouteConfig.php

```diff
@@ -45,19 +45,19 @@ final class RouteConfig
     private $defaults;
 
     /**
-     * @var array
+     * @var array<string>|string
```

Comment:
```suggestion
     * @var string|array<string>
```

Reply:
fixed in 7cc0debe1e3549a1743a89fb9a87475285d769b4. It's actually always an array, but the constructor argument is not.